tests/k8s/tests/03-l7-stresstest.sh

```diff
@@ -89,12 +89,14 @@ backend_svc_ip=$(kubectl get svc -n development | awk 'NR==2{print $2}')
 
 log "Running tests WITHOUT Policy / Proxy loaded"
 
+log "running curl ${backend_svc_ip}:80 from pod ${frontend_pod} (should work)"
 code=$(kubectl exec -n qa -i ${frontend_pod} -- curl -s -o /dev/null -w "%{http_code}" http://${backend_svc_ip}:80/)
 
 if [ ${code} -ne 200 ]; then abort "Error: unable to connect between frontend and backend:80/" ; fi
 
-kubectl exec -n qa -i ${frontend_pod} -- wrk -t20 -c1000 -d60 "http://${backend_svc_ip}:80/"
-kubectl exec -n qa -i ${frontend_pod} -- ab -r -n 1000000 -c 200 -s 60 -v 1 "http://${backend_svc_ip}:80/"
+#log "running stress tests"
```

Comment:
Please file a GitHub issue and add `GH-XXX` comment to make sure we don't forget to re-enable this.